src/drugs.cpp

```diff
@@ -412,6 +416,13 @@ void apply_drug_modifiers_to_ch(struct char_data *ch) {
           GET_TARGET_MOD(ch)++;
           GET_WIL(ch)++;
           break;
+        case DRUG_DEEPWEED:
+          GET_WIL(ch)++;
+          if (GET_TRADITION(ch) == TRAD_HERMETIC || GET_TRADITION(ch) == TRAD_SHAMANIC || GET_TRADITION(ch) == TRAD_ADEPT)
+          PLR_FLAGS(ch).SetBit(PLR_PERCEIVE);
+          else if (GET_TRADITION(ch) != TRAD_MUNDANE)
+          snprintf(buf, sizeof(buf), "Your vision blurs but it fails to pierce the astral veil.\r\n");
```

Comment:
Have you tested this? I think it's gonna spam mundies since this function is called relatively often.